quickstart/101-virtual-network-manager-create-mesh/main.tf

```diff
@@ -58,14 +58,56 @@ resource "azurerm_network_manager_network_group" "network_group" {
   network_manager_id = azurerm_network_manager.network_manager_instance.id
 }
 
-# Add the three virtual networks to the network group as static members
+# Add three virtual networks to a network group as dynamic members with Azure Policy
 
-resource "azurerm_network_manager_static_member" "static_members" {
-  count = 3
+resource "random_pet" "network_group_policy_name" {
+  prefix = "network-group-policy"
+}
+
+resource "azurerm_policy_definition" "network_group_policy" {
+  name         = "${random_pet.network_group_policy_name.id}"
+  policy_type  = "Custom"
+  mode         = "Microsoft.Network.Data"
+  display_name = "Policy Definition for Network Group"
+
+  metadata = <<METADATA
```

Comment:
Could we rewrite this heredoc string with a Terraform native object and [`jsonencode`](https://developer.hashicorp.com/terraform/language/functions/jsonencode) function?

Reply:
I'll pass this on to the Product Team, and see if an engineer is able to put this together.

Reply:
This is not something the PG needs to be aware of / change but I think retaining the Azure Policy format in this case is probably better.